tests/Nelmio/Alice/Fixtures/Parser/Methods/PhpTest.php

```diff
@@ -110,4 +110,31 @@ public function testIncludeFiles()
         ];
         $this->assertEquals($expectedData, $data);
     }
+
+    public function testIncludeFilesOrder()
+    {
+        $data = $this->parser->parse(__DIR__.'/../../../support/fixtures/include_order.php');
+
+        $expectedData = [
+            'Foo' =>
+                [
+                    'foo' =>
+                        [
+                            'id' => 1,
+                            'text' => '<word()>',
+                        ],
+                ],
+            'Bar' =>
+                [
+                    'bar' =>
+                        [
+                            'id' => 1,
+                            'text' => '<@foo->text>'
+                        ],
+                ],
+        ];
+
+        $this->assertEquals($expectedData, $data);
+        $this->assertTrue($expectedData === $data);
```

Comment:
>but it is useful to get the differences in output.

What do you mean? In this case they do exactly the same thing as they are arrays and not objects, the only difference is that `===` will make sure they are of the same type, hence `array` here. As you're giving the expected result (which is a non-empty array), this check is unneeded.